app/actions/websocket.ts

```diff
@@ -1126,8 +1140,18 @@ function handleOpenDialogEvent(msg: WebSocketMessage) {
 }
 
 // Helpers
-export function notVisibleUsersActions(state: GlobalState): Array<GenericAction> {
-    const knownUsers = getKnownUsers(state);
+export async function notVisibleUsersActions(state: GlobalState): Promise<Array<GenericAction>> {
+    if (isMinimumServerVersion(Client4.getServerVersion(), 5, 23)) {
```

Comment:
The check is here

Reply:
this is actually the other way around right? if the server is 5.23 or above it executes the block inside the if

Reply:
:facepalm: Too many things at the same time. Really sorry.

Reply:
changed